pkg/ingress/config_types.go

```diff
@@ -88,6 +88,10 @@ func (t *TargetGroupTuple) validate() error {
 	if t.ServiceName != nil && t.ServicePort == nil {
 		return errors.New("missing servicePort")
 	}
+
+	if (t.Weight != nil) && (*(t.Weight) < 0 || *(t.Weight) > 999) {
+		return errors.New("The weight for each target group must be between 0 and 999, inclusive")
```

Comment:
Error messages should not start with a capital letter.

Reply:
accepted